torch/csrc/cuda/Module.cpp

```diff
@@ -150,7 +150,7 @@ PyObject * THCPModule_setRNGState(PyObject *_unused, PyObject *obj)
     throw TypeError("set_rng_state expects a torch.ByteTensor, but got %s",
         Py_TYPE(obj)->tp_name);
   }
-  auto& tensor = THPVariable_UnpackData(obj);
+  auto& tensor = THPVariable_Unpack(obj);
```

Comment:
The original use of `THPVariable_UnpackData()` here is to make sure that the `THCRandom_setRNGState(state, (THByteTensor*)tensor.unsafeGetTensorImpl())` call below is using the Variable's underlying `data_`'s TensorImpl instead of the Variable's `Variable::Impl`. Since now there is no `data_` and a Variable stores its tensor data directly in its TensorImpl, and the `THCRandom_setRNGState(...)` call doesn't care about whether the TensorImpl has autograd metadata, it's okay to remove `THPVariable_UnpackData()` and use `THPVariable_Unpack()` here.